src/V3Error.h

```diff
@@ -67,6 +67,7 @@ class V3ErrorCode final {
         E_PORTSHORT,    // Error: Output port is connected to a constant, electrical short
         E_TASKNSVAR,    // Error: Task I/O not simple
         E_UNSUPPORTED,  // Error: Unsupported (generally)
+        HIERPARAM,      // Error: Parameter using hierarchical value
```

Comment:
I believe this is supposed to have an `E_` prefix if it's back to being a for-real error.